app/services/pipeline_service/V2/client.rb

```diff
@@ -0,0 +1,21 @@
+module PipelineService
+  module V2
+    class Client
+      REGION = 'us-west-2'
```

Comment:
We've got this stored in an environment variable now, so we should probably keep that consistent: `REGION = ENV['AWS_REGION'] || 'us-west-2'`